br/pkg/restore/data.go

```diff
@@ -281,8 +309,41 @@ func (recovery *Recovery) RecoverRegions(ctx context.Context, allStores []*metap
 				log.Error("close the stream failed")
 				return errors.Trace(err)
 			}
+			recovery.progress.Inc()
+			log.Info("recovery command execution success", zap.Uint64("storeID", reply.GetStoreId()))
+			return nil
+		})
+	}
+	// Wait for all TiKV instances force leader and wait apply to last log.
+	return eg.Wait()
+}
+
+// send wait apply to all tikv ensure all region peer apply log into the last
+func (recovery *Recovery) WaitApply(ctx context.Context, allStores []*metapb.Store) (err error) {
+	eg, ectx := errgroup.WithContext(ctx)
+	totalStores := len(recovery.recoveryPlan)
```

Comment:
ditto

Reply:
Yes, waitapply shall send to all region, so it should be len(allStores).